routes/web.php

```diff
@@ -36,6 +37,7 @@
 Route::middleware('auth')->group(function () {
     //Insert here every route that need user login
     Route::get('/decision', [DecisionController::class, 'index']);
+    Route::get('/professionalFields', [DecisionController::class, 'showDecisionPage']);
```

Comment:
That route name does not fit. We already have a /decision route to handle that decision process. So pls override the route above with your function

Reply:
changed